### PR TITLE
upgrade io-spid-commons and configure Contact Person

### PR DESCRIPTION
**Pull request by AleDore**
This PR upgrades io-spid-commons to the latest version in order to enable and configure spid to expose metadata as a `PublicServicesFullOperator`

### CODE REVIEW

**Review by BurnedMarshal — CHANGES_REQUESTED** [review]

File: src/utils/config.ts [hunk 1]
```diff
@@ -114,6 +151,9 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   ENABLE_ADE_AA: fromNullable(process.env.ENABLE_ADE_AA)
     .map(_ => _.toLowerCase() === "true")
     .getOrElseL(() => false),
+  ENABLE_FULL_OPERATOR_METADATA: fromNullable(process.env.ENABLE_ADE_AA)
```
[the remark this answers is not in the document]
Comment (AleDore) [reply src/utils/config.ts]: You're right 😅

File: src/utils/config.ts [hunk 2]
```diff
@@ -114,6 +151,9 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   ENABLE_ADE_AA: fromNullable(process.env.ENABLE_ADE_AA)
     .map(_ => _.toLowerCase() === "true")
     .getOrElseL(() => false),
+  ENABLE_FULL_OPERATOR_METADATA: fromNullable(process.env. ENABLE_FULL_OPERATOR_METADATA)
```
Comment (BurnedMarshal) [inline src/utils/config.ts:154]: Sorry, a whitespace typo in the previous code suggestion.
```suggestion
  ENABLE_FULL_OPERATOR_METADATA: fromNullable(process.env.ENABLE_FULL_OPERATOR_METADATA)
```

Comment (BurnedMarshal) [reply src/utils/config.ts:154]: Take a look at the linter error in the review pipeline.

**Review by BurnedMarshal — APPROVED** [review]
LGTM

**Pull request merged**